Core/src/org/sleuthkit/autopsy/casemodule/events/HostsAddedToPersonEvent.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2021 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.casemodule.events;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.sleuthkit.autopsy.casemodule.Case;
+import org.sleuthkit.datamodel.Host;
+import org.sleuthkit.datamodel.Person;
+import org.sleuthkit.datamodel.SleuthkitCase;
+import org.sleuthkit.datamodel.TskCoreException;
+
+/**
+ * Application events published when one or more hosts have been added to a
+ * person.
+ */
+public final class HostsAddedToPersonEvent extends TskDataModelChangedEvent<Person, Host> {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Constructs an application event published when one or more hosts have
+     * been added to a person.
+     *
+     * @param person The person.
+     * @param hosts  The hosts.
+     */
+    public HostsAddedToPersonEvent(Person person, List<Host> hosts) {
+        super(Case.Events.HOSTS_ADDED_TO_PERSON.toString(), Collections.singletonList(person), Person::getPersonId, hosts, Host::getHostId);
+    }
+
+    /**
+     * Gets the person.
+     *
+     * @return The person.
+     */
+    public Person getPerson() {
+        return getOldValue().get(0);
+    }
+
+    /**
+     * Gets the hosts.
+     *
+     * @return The hosts.
+     */
+    public List<Host> getHosts() {
+        return getNewValue();
+    }
+
+    @Override
+    protected List<Person> getOldValueObjects(SleuthkitCase caseDb, List<Long> ids) throws TskCoreException {
+        List<Person> persons = new ArrayList<>();
```

Comment:
For this class, `HostsAddedToPersonEvent`, and the `HostsRemovedFromPersonEvent` class, would it make sense for the old and new values to relate to the list of `Host` objects (ids or objects)?  I initially had a little confusion that the 'new values' were the child `Host`s and the 'old value' was the parent `Person` object.